dist/jquery.gridster.js

```diff
@@ -4587,7 +4589,7 @@
 	 */
 	fn.get_responsive_col_width = function () {
 		var cols = this.cols || this.options.max_cols;
-		return (this.$el[0].scrollWidth - ((cols + 1) * this.options.widget_margins[0])) / cols;
+		return (this.$el[0].clientWidth - 3 - ((cols + 1) * this.options.widget_margins[0])) / cols;
```

Comment:
Honestly?  This is a really bad case of using a "magic number" and I have completely forgotten.  This PR had a couple of small issues that got it working quickly for a project but caused some other issues and I never got around to trying to fix the other issues it caused (we moved away from Gridster).